app/fragments/HomeFragment.tsx

```diff
@@ -43,7 +43,11 @@ export const HomeFragment = fragment(() => {
                             }
 
                             if (existing.job.job.type === 'transaction') {
-                                SplashScreen.hideAsync();
+                                try {
+                                    SplashScreen.hideAsync();
```

Comment:
Is this the only place there splash screen is hidden?

Reply:
No, this one is to ensure that native gets hidden in case of opening Transfer Fragment with QR code scanned.